### ISSUE TITLE
Error : Class 'ChargeBee\ChargeBee\Models\Exception' not found

### ISSUE THREAD

**Issue by sirber** [chargebee/chargebee-php#60]
### Description of the Bug

in `Models\Event.php`, in deserialize(), there's a `throw new Exception`

### Steps to reproduce

call `deserialize()` with `$json = "{}"`

### Expected Behavior

a `\Exception` is thrown

### Code Snippets (if applicable)

```PHP
<?php

namespace ChargeBee\ChargeBee\Models;

use ChargeBee\ChargeBee\Model;
use ChargeBee\ChargeBee\Request;
use ChargeBee\ChargeBee\Util;
use ChargeBee\ChargeBee\Environment;

class Event extends Model
{
    public static function deserialize($json)
    {
        $webhookData = json_decode($json, true);
        if(!$webhookData) {
            throw new Exception("Response not in JSON format. Might not be a ChargeBee webhook call.");
        }
```

### Operating System

Linux

### Language version

PHP 7.4

### Library version

v3.31.0

**Comment by sirber** [comment]
I ran a patch and it works now.

![image](https://github.com/user-attachments/assets/705ed3d3-fe7e-43bb-be3d-d99e8364982d)

**Comment by sirber** [comment]
been fixed in 3.32.0

https://github.com/chargebee/chargebee-php/compare/v3.31.0...v3.32.0
